_sass/_basic.scss

```diff
@@ -328,3 +328,43 @@ ul > li {
 .fill-screen-row {
     width: 100%;
 }
+
+.resource-logo {
+    max-height: 200px;
+    max-width: 200px;
+    width: auto;
+    height: auto;
+    margin: auto;
+    @media (max-width: $desktop-width) {
+       width: 90%;
+   }
+}
+
+.resource-description {
+    align-items: left;
+    margin-right: 1em; 
+    margin-left: 1em;         
+}

```

Comment:
I think this class isn't necessary - you can instead add padding to the parent container?